src/snippets/responsive-bg-image.liquid

```diff
@@ -0,0 +1,39 @@
+{% comment %}
+    It creates a style tag containing "background-image" properties to load the correct image depending on the resolution
+    Dependencies:
+    - Lazysizes plugin (https://github.com/aFarkas/lazysizes) which enable responsive image with faster load time and better performance.
+    - Lazysizes Bgset (https://github.com/aFarkas/lazysizes/tree/gh-pages/plugins/bgset) To use responsive images on background-image (CSS)
+
+    Accepts:
+    - image: {Object} Image object
+
+    Usage:
+    In your liquid template file, copy the following line
+
+    <div class="responsive-bg-image lazyload" data-bgset="{% include 'responsive-bg-image', image: article.image %}"></div>
+
+    - It must include "responsive-bg-image lazyload" wrapper element
```

Comment:
How come it has to include `responsive-bg-image`? I didn't see mention of that class in [the plugin docs](https://github.com/aFarkas/lazysizes/tree/gh-pages/plugins/bgset) and I don't see any CSS associated with it.

I believe you only need `data-bgset` to get this to work.

Reply:
My mistake, it was from the initial PR and some changes were made so that class is not needed anymore.